torch/optim/swa_utils.py

```diff
@@ -84,7 +84,7 @@ class AveragedModel(Module):
         Generalizes Well:
         https://arxiv.org/abs/2001.02312
     """
-    def __init__(self, model, device=None, avg_fn=None):
+    def __init__(self, model, device=None, avg_fn=None, use_state_dict=False):
```

Comment:
Also the parameter `use_state_dict` needs to be added in the documentation of the class.

Reply:
Added.